src/usr/share/netcheck/netcheck-daemon.py

```diff
@@ -96,19 +97,21 @@ def sig_term_handler(signal, stack_frame):
     daemon_context = daemon.DaemonContext(
         working_directory='/',
         pidfile=pidlockfile.PIDLockFile(PID_FILE),
-        umask=0
+        umask=PROGRAM_UMASK
         )
 
     daemon_context.signal_map = {
         signal.SIGTERM: sig_term_handler
         }
 
     daemon_context.files_preserve = [log_file_handle]
+    broadcaster = broadcaster.Broadcaster(
```

Comment:
Can you do me a favor and add a TODO here to update this to netcheck user and group IDs.